hubmap/portal/tests.py

```diff
@@ -0,0 +1,37 @@
+from django.test import RequestFactory, TestCase
+
+# Create your tests here.
```

Comment:
Now that you have tests, this message is unneeded.

Reply:
will do.

Reply:
removed.